copy/fixtures

```diff
@@ -0,0 +1 @@
+../pkg/compression/fixtures
```

Comment:
Using symlinks to deduplicate is fine, but could you please keep the `copy/fixtures` directory and use symlinks to individual files? That would make it easier to add other files to `copy/fixtures` in the future.